server/routes/api/ranking.py

```diff
@@ -31,28 +35,55 @@ def ranking_api(stat_var, place_type, place=None):
     """Returns top 100 rankings for a stats var, grouped by place type and
```

Comment:
update the comments here, and might add some explanation on the different scenarios for top, bottom and all places

Reply:
Done